2017/03/01 10:29:54 [error] 1918#0: *101841 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.15.85, server: , request: "GET /LBS02/services/postions/861201033875297/2017-02-28%2000%3A00%3A00/2017-02-28%2024%3A00%3A00 HTTP/1.1", upstream: "http://192.168.15.56:9904/LBS02/services/postions/861201033875297/2017-02-28%2000%3A00%3A00/2017-02-28%2024%3A00%3A00", host: "api.vats.com.cn:9904"


2017/03/01 10:30:20 [error] 1918#0: *101957 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.15.107, server: , request: "GET /LBS02/services/postions/867469026659366/2017-02-06%2000:00:00/2017-02-06%2024:00:00 HTTP/1.1", upstream: "http://192.168.15.56:9904/LBS02/services/postions/867469026659366/2017-02-06%2000:00:00/2017-02-06%2024:00:00", host: "api.vats.com.cn:9904"
2017/03/01 10:31:16 [error] 1918#0: *102257 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.15.85, server: , request: "GET /LBS02/services/postions/8682270258660721/2017-03-01%2000%3A00%3A00/2017-03-01%2024%3A00%3A00 HTTP/1.1", upstream: "http://192.168.15.56:9904/LBS02/services/postions/8682270258660721/2017-03-01%2000%3A00%3A00/2017-03-01%2024%3A00%3A00", host: "api.vats.com.cn:9904"
2017/03/01 10:34:26 [error] 1916#0: *103043 open() "/usr/local/nginx/html/favicon.ico" failed (2: No such file or directory), client: 192.168.17.73, server: , request: "GET /favicon.ico HTTP/1.1", host: "api.vats.com.cn:9904", referrer: "http://api.vats.com.cn:9904/workInfoService/workinfo/3EA0CF10-FB39-11E2-8F10-C7A1C15F563D/0E2391E0-FB4A-11E2-91E0-99DCC208307A/2017-03-01"
2017/03/01 10:35:34 [error] 1916#0: *103350 open() "/usr/local/nginx/html/favicon.ico" failed (2: No such file or directory), client: 192.168.17.95, server: , request: "GET /favicon.ico HTTP/1.1", host: "api.vats.com.cn:9904", referrer: "http://api.vats.com.cn:9904/workInfoService/workinfo/3EA0CF10-FB39-11E2-8F10-C7A1C15F563D/0E2391E0-FB4A-11E2-91E0-99DCC208307A/2017-03-01"
